src/widgets/favoritewidget.cpp

```diff
@@ -22,18 +22,18 @@
 #include <QSize>
 #include <QStyle>
 #include <QStylePainter>
+#include <QFontMetrics>
 
 #include "core/logging.h"
 
-const int FavoriteWidget::kStarSize = 15;
+const int FavoriteWidget::kStarSize = 16;
 
 FavoriteWidget::FavoriteWidget(int tab_index, bool favorite, QWidget* parent)
     : QWidget(parent),
       tab_index_(tab_index),
       favorite_(favorite),
       on_(":/star-on.png"),
-      off_(":/star-off.png"),
-      rect_(0, 0, kStarSize, kStarSize) {}
```

Comment:
what about this variable in favoritewidget.h ? any way i don't think deleting this variable is a good idea..